src/components/Blocking/ReportDomain.js

```diff
@@ -113,7 +114,7 @@ const styles = StyleSheet.create({
   title: {
     fontFamily: fonts.inter[700],
     fontSize: 16,
-    color: '#000',
+    color: COLORS.black,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This change also improves maintainability and consistency by replacing the hard-coded color with a value from the theme.
```diff
- color: '#000',
+ color: COLORS.black,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->